lib/internal/repl/await.js

```diff
@@ -11,6 +11,12 @@ const visitorsWithoutAncestors = {
     }
     walk.base.ClassDeclaration(node, state, c);
   },
+  ForOfStatement(node, state, c) {
```

Comment:
nit: for parity with [Chromium code](https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/formatter_worker/FormatterWorker.js?q=formatter_worker/FormatterWorker.js&sq=package:chromium&g=0&l=188), I would move this to right after `AwaitExpression`.